docs/getting-started-guides/vagrant.md

```diff
@@ -412,6 +412,24 @@ export KUBERNETES_MASTER_MEMORY=1536
 export KUBERNETES_NODE_MEMORY=2048
 ```
 
+#### I want to set proxy settings for my Kubernetes cluster boot strapping!
+
+If you are behind a proxy, you need to install vagrant proxy plugin and set the proxy settings by
```

Comment:
minor suggestion: could maybe link to https://github.com/tmatilai/vagrant-proxyconf or include `vagrant plugin install vagrant-proxyconf`?